src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -14,6 +14,7 @@ import ProcessStore from '~app/common/stores/applications/SsvWeb/Process.store';
 import NewWhiteWrapper from '~app/components/common/NewWhiteWrapper/NewWhiteWrapper';
 import NotificationsStore from '~app/common/stores/applications/SsvWeb/Notifications.store';
 import OperatorStore, { IOperator } from '~app/common/stores/applications/SsvWeb/Operator.store';
+import { CopyButton } from '~app/components/common/Button/CopyButton/CopyButton';
```

Comment:
In all the files that you touched sort imports using these rules:

1. Sort all third-party imports from shorter import to longer
2. Sort all the app-specific imports from shorter to longer
3. Importing React should be always first regardless length and type of import

Reply:
done